### PR TITLE
Add GH Discussions badge

### PR DESCRIPTION
**Pull request by armab**
A tiny cosmetic change that adds a badge with a counter, linking to BitOps Discussions.
image

### CODE REVIEW

**Review by mickmcgrath13 — APPROVED** [review]

**Review by ccapell — APPROVED** [review]

**Review by ccapell — CHANGES_REQUESTED** [review]
Please wait until #272 is merged.  Then this can move foward

**Review by ccapell — APPROVED** [review]

**Pull request merged**
